dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -180,16 +181,6 @@ const LiveGrid = ({ children }: { children: React.ReactNode }) => (
 				}
 				/* until desktop define fixed body width */
 				${until.desktop} {
-					grid-template-columns: 700px; /* Main content */
-					grid-template-areas:
-						'media'
-						'lines'
-						'meta'
-						'keyevents'
-						'body';
-				}
-				/* fluid until tablet */
-				${until.tablet} {
```

Comment:
Yes! This was redundant!

Just one thing, does this mean `700px` can be `1fr` now?

Reply:
Yes - I suppose it can as the content is full width on breakpoints up to desktop. Good point Oliver!